schema.json

```
@@ -1,329 +1,471 @@
+      "type": "string",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "description_longue",
+      "title": "Description longue",
+      "description": "Description longue du dispositif - facultatif, texte enrichi en markdown. Elle peut être ajoutée si on juge que l'utilisateur ne dispose pas des informations suffisantes avec les seules autres informations. Elle peut être ajoutée par exemple :\n- pour définir un terme technique\n- pour détailler une loi à laquelle un dispositif fait référence\n- pour détailler les projets concernés par un financement\n\nFormulation :\n- 50 à 200 mots",
+      "example": "L'étude ACT Pas à Pas se déroule en 12 à 18 mois selon les entreprises. Les grandes étapes de l'étude sont les suivantes :\n- Analyser la maturité initiale de l'entreprise sur des bases carbone.\nExposer les enjeux et réaliser une analyse stratégique de l’entreprise dans un contexte de transition bas carbone.\n- Déterminer une vision d’entreprise et concevoir la feuille de route du plan de transition à court, moyen et long terme.\n- Elaborer des objectifs de performance carbone et définir une stratégie.\n- Mettre en oeuvre des actions de décarbonation.",
+      "type": "string"
+    },
+    {
+      "name": "programmes_aide",
+      "title": "Programme(s) auquel se réfère l'aide",
+      "description": "Programme(s) auquel se réfère l'aide - facultatif. Si aucun programme, ne pas remplir ce champs. S'il y a plusieurs programmes, mettre un pipe (|) sans espace entre les différents programmes.",
+      "example": "France 2030|Fonds social européen (FSE)|Fonds Vert",
+      "type": "string"
+    },
+    {
+      "name": "porteurs_aide",
+      "title": "Porteur(s) de l'aide",
+      "description": "Porteur(s) de l'aide. S'il y a plusieurs porteurs, mettre un pipe (|) sans espace entre les différents porteurs. Le premier porteur est considérer le porteur/opérateur principal de l'aide.",
+      "example": "ADEME|Région Grand Est",
+      "type": "string",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "porteurs_siren",
+      "title": "SIREN du.es porteur.s",
+      "description": "Numéro SIREN du/des porteur(s) de l'aide. S'il y a plusieurs porteurs, mettre un pipe (|) sans espace entre les différents porteurs.",
+      "example": "385290309|483790069",
+      "type": "string",
+      "constraints": {
+        "required": false,
+        "pattern": "^(?:\\d{9}\\|)*\\d{9}$"
+      }
+    },
+    {
+      "name": "beneficiaires_aide",
+      "title": "Bénéficiaire.s de l'aide",
```

Pour être cohérent avec les formulations précédentes
```suggestion
      "title": "Bénéficiaire(s) de l'aide",
```